src/config-local.tsx

```diff
@@ -57,5 +57,6 @@ export default {
     { "parachainName":'Bifrost (coming soon...)', "parachainId": 0,'isDisabled':true },
     { "parachainName":'Kusama (coming soon...)', "parachainId": 0, 'isDisabled':true },
     { "parachainName":'Snowbridge (coming soon...)', "parachainId": 0,'isDisabled':true }
-  ]
+  ],
+  TRANSACTION_FEES:'0'
```

Comment:
Fees would be a different per parachain. So it needs to be part of `PARACHAIN_LIST` lists parachain data.

Reply:
As suggested, I've updated the changes.